### PR TITLE
Add macos DNS alias

### PR DESCRIPTION
**Pull request by nunorafaelrocha**
This PR adds macos alias to manage DNS servers for wi-fi network.

### CODE REVIEW

File: macos/set-defaults.sh [hunk 1]
```diff
@@ -58,6 +58,13 @@ defaults write com.apple.menuextra.battery ShowPercent YES
 #sudo rm -rf /System/Library/CoreServices/DefaultDesktop.jpg
 #sudo ln -s /path/to/your/image /System/Library/CoreServices/DefaultDesktop.jpg
 
+###############################################################################
+# Network                                                                     #
+###############################################################################
+
+# Setup Google and Cloudflare DNS servers as Wi-Fi network DNS servers.
```
Comment (LankyLou) [inline macos/set-defaults.sh]:
```suggestion
# Setup Google & Cloudflare DNS servers as Wi-Fi network DNS servers.
```

File: macos/set-defaults.sh [hunk 2]
```diff
@@ -58,6 +58,13 @@ defaults write com.apple.menuextra.battery ShowPercent YES
 #sudo rm -rf /System/Library/CoreServices/DefaultDesktop.jpg
 #sudo ln -s /path/to/your/image /System/Library/CoreServices/DefaultDesktop.jpg
 
+###############################################################################
+# Network                                                                     #
+###############################################################################
```
Comment (vvnkr) [inline macos/set-defaults.sh]: I think you're missing a `#` here.

Comment (nunorafaelrocha) [reply macos/set-defaults.sh]: no

**Comment by nunorafaelrocha** [comment]
> I don't like these changes

Why?

File: macos/set-defaults.sh (hunk 1 above)
Comment (nunorafaelrocha) [reply macos/set-defaults.sh]: no

**Pull request merged**